Changelog.md

```diff
@@ -1,5 +1,10 @@
 ## libsdformat 10.X
 
+### libsdformat 10.5.0 (2021-05-17)
+
+1. Add scatter ratio parameter to Particle Emitter DOM
+    + [Pull request 547](https://github.com/osrf/sdformat/pull/547)
```

Comment:
Let's add changelog for #521  and #555 as well

Reply:
c60384f5

Reply:
thanks!